data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/consumer/OrderedConsumerVerticle.java

```diff
@@ -104,7 +95,7 @@ public OrderedConsumerVerticle(final DataPlaneContract.Egress egress,
   @Override
   void startConsumer(Promise<Void> startPromise) {
     // We need to sub first, then we can start the polling loop
-    this.consumer.subscribe(this.topics)
+    this.consumer.subscribe(new HashSet<>(getConsumerVerticleContext().getResource().getTopicsList()))
```

Comment:
I am not a fan of inline ctors, could we use 

`Set.copyOf(getConsumerVerticleContext().getResource().getTopicsList())` ?

Reply:
Let me try, didn't know about that API :eyes:

Reply:
Done